Sources/SwiftSentry/User.swift

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: BSD-3-Clause
+import Foundation
```

Comment:
Why this import?

Reply:
Think I just goofed when I was writing this, happy to remove